cli/replicaset/roles.go

```diff
@@ -48,24 +87,32 @@ type RolesChangeCtx struct {
 	Timeout int
 }
 
-// RolesAdder is an interface for adding roles to a replicaset.
-type RolesAdder interface {
-	// RolesAdd adds role to a replicasets by its name.
-	RolesAdd(ctx RolesChangeCtx) error
+// RolesChanger is an interface for adding/removing roles to a replicaset.
+type RolesChanger interface {
+	// RolesChange adds/remove role to a replicasets by its name.
+	RolesChange(ctx RolesChangeCtx, changeRoleAction RolesChangerAction) error
```

Comment:
The same.
```suggestion
	RolesChange(ctx RolesChangeCtx, action RolesChangerAction) error
```
Please, apply the change to the same code in the pull request.

Reply:
Renamed.